### ISSUE TITLE
Be able to archive a groupfolder

### ISSUE THREAD

**Issue by rassakali** [nextcloud/groupfolders#1156]
Hello,

Is there a way to archive a groupfolder? that is to say, an archived groupfolder, remove it from the main view but it remains consultable via another menu item or group them together in an archives folder.
For internal reasons we create a groupfolder work by project. we want to archive the groupfolder once the project is finished. because now we find ourselves with a hundred groups that we consult very rarely.

Thank you in advance for your answers

**Comment by Chartman123** [comment]
Couldn't you just create a top-level groupfolder "Archive" and move the archived groupfolders into that parent folder by changing their names to "Archive/groupfolder-name"?

**Comment by rassakali** [comment]
Thank you very much for your answer.

I don't think you can move one groupfolder to another.
On the other hand, we can move the content of one groupfolder to another, I think you suggest me?

Know that each groupfolder has different users and accesses. In this case, if I move the content into the groupfolder archive, the users lose access to their documents. unless I give the Archive group access to everyone, that's what we don't want.

I don't know if we can ask the community to add this feature?

**Comment by Chartman123** [comment]
Yes, you can move the whole folder the way I wrote in  my first post:

Create three folders, one as a subfolder of another:
![image](https://user-images.githubusercontent.com/16020496/100423157-d9f17700-308b-11eb-8cbb-c6d5d93ff9cc.png)
Create a file in the subfolder
![image](https://user-images.githubusercontent.com/16020496/100423221-f1306480-308b-11eb-88d2-af6e5652e39f.png)

Edit the name of the subfolder and put in the second folder:
![image](https://user-images.githubusercontent.com/16020496/100423285-060cf800-308c-11eb-91ea-2672d0da05c6.png)
File and subfolder appear in the second folder:
![image](https://user-images.githubusercontent.com/16020496/100423307-11602380-308c-11eb-9fd5-e4aa6857a9e0.png)

**Comment by rassakali** [comment]
Thank you for your answer.

Yes I know, but it doesn't work if I want to move "Temp" into Temp2. 

look at the following files:
![Image 8](https://user-images.githubusercontent.com/16742126/100747118-ee15db00-33e1-11eb-869f-cf6a04d98953.png)

It's not possible to move the "GroupeFolder1" inside "Archives", because "GroupeFolder1" is a folder mounted by groupfolder app.

The need is to remove from first page of files, the groupfolders that are no longer relevant.

**Comment by Chartman123** [comment]
Yes, I know that... So just go into your groupfolder settings and change the name of GroupFolder1 to Archives/GroupFolder1... By changing it in the settings it will be moved under the Archives folder in the front-end

**Comment by ZeikoFr** [comment]
Hi @rassakali I can confirm that renaming your folder Archives/GroupFolder1 will move the group folder correctly and will keep the file.

Of course I recommend you to make a backup of the server before making the change.

Regards

**Comment by pierreozoux** [comment]
The questoion wan answered, Ill close the issue.
